packages/onboarding/src/Client.ts

```diff
@@ -0,0 +1,66 @@
+import axios, { AxiosInstance, AxiosResponse, Method } from 'axios'
+import qs from 'qs'
+import { OnboardingRecord, Maker, DEWI_ONBOARDING_API_BASE_URL } from './types'
+
+type Response<T>={
+  code: number
+  data: T | null
+  success: boolean
+  errorMessage?: string
+  errors?: Array<any>
+}
+
+export default class Client {
```

Comment:
could having this top level class named `Client` get confusing with the `Client` exported out of `@helium/http`?

Reply:
Likely. I'll rename.